CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 **Improvements:**
 
 * `resource/cloudflare_access_policy`: Add support for `service_token` and `any_valid_service_token` ([#612](https://github.com/terraform-providers/terraform-provider-cloudflare/issues/612))
+* `resource/cloudflare_access_policy`: Add support for `group` ([#188
```

Comment:
You can omit this; it's added post merge by the maintainers.

Reply:
Ok, so I reverted here